### ISSUE TITLE
[pimatic] install specific version of plugin

### ISSUE THREAD

**Issue by michbeck100** [pimatic/pimatic#1139]
As a pimatic user I want to be able to install a specific version of a plugin instead of the latest version. 

this could be implemented by adding the version to the plugin name in the `config.json`, like

```
  "plugins": [
    {
      "plugin": "cron@0.8.7"
    }
  ]
```

**Comment by bertreb** [comment]
Hi, nice feature to allow users to stays on a working stable version of a plugin. Are the dependencies also automatically frozen?

**Comment by michbeck100** [comment]
It's the same behavior as if you would install without fixed versions. It always just installs the dependencies that are needed for the specific version. If the version is omitted it will just use the latest version, that's all. So the answer is yes

**Comment by bertreb** [comment]
Ok, but if dependencies develop and in the future don't work the 'frozen' main code anymore, the plugin stops working. Maximising dependency versions in a the package version could be a solution. This should be a good habbit of a plugin developer 😃

**Comment by michbeck100** [comment]
This depends one the plugin itself. If the developer doesn't use fixed versions for the dependencies there's not much you can do.

**Comment by bertreb** [comment]
Agree, thats why it should be clear to people using fix plugin version, that there are no guarantees in the future that the plugin keeps working. On reinstall of Pimatic or node upgrade this could happen. If they don't touch the system anymore it's fine.
But the big advantage of this change is that you can upgrade all other plugins and pimatic 👍

**Comment by michbeck100** [comment]
Since #1140 was merged, when will be the next pimatic release? The newest pimatic-hap update needs at least nodejs 8. So users who want to stick to older nodejs versions need this change.

**Comment by michbeck100** [comment]
Also https://github.com/pimatic/pimatic/pull/1138 would help a lot with this.
